packages/babel-plugin-proposal-pipeline-operator/src/minimalVisitor.ts

```diff
@@ -1,7 +1,8 @@
-import { types as t } from "@babel/core";
+import { types as t, type PluginObject } from "@babel/core";
+import type { NodePath } from "@babel/traverse";
 import buildOptimizedSequenceExpression from "./buildOptimizedSequenceExpression";
 
-const minimalVisitor = {
+const minimalVisitor: PluginObject["visitor"] = {
```

Comment:
Why don't we use `Visitor` from `@babel/traverse`?

Reply:
No particular reason. Guess at that point I was lazy to add the Visitor import, and we have to add `PluginPass`, too. Maybe we can export `PluginVisitor = Visitor<PluginPass>` from `@babel/core`?